theme_vehicle/views/customizations.xml

```diff
@@ -1,171 +1,365 @@
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="o_cc o_cc1 pt120 pb112" remove="pt48 pb32" separator=" "/>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/05","flip":[]}</attribute>
+        </xpath>
+        <xpath expr="//div[hasclass('container')]" position="before">
+            <div class="o_we_shape o_web_editor_Origins_05"/>
+        </xpath>
+        <xpath expr="//h2" position="replace" mode="inner">
+            Our Partners
+        </xpath>
+        <xpath expr="//p" position="replace"/>
+    </template>
+
+     <!-- ======== CALL TO ACTION ======== -->
+    <template id="s_call_to_action" inherit_id="website.s_call_to_action" name="Vehicle s_call_to_action">
+        <!-- Section -->
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="o_cc5 pt96 pb64" remove="o_cc3 pt48 pb24" separator=" "/>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Floats/09","flip":[]}</attribute>
+        </xpath>
+        <!-- Shape -->
+        <xpath expr="//div[hasclass('container')]" position="before">
+            <div class="o_we_shape o_web_editor_Floats_09"/>
+        </xpath>
+        <!-- Content -->
+        <xpath expr="//h3" position="replace" mode="inner">
+            <b>50,000 pre-orders</b> already registered.
+        </xpath>
+        <xpath expr="//h3" position="attributes">
+            <attribute name="style" add="text-align: left;" separator=" "/>
+        </xpath>
+        <xpath expr="//p" position="replace" mode="inner">
+            Join the early joiners or request for a trial on track.
+        </xpath>
+        <xpath expr="//p" position="attributes">
+            <attribute name="style" add="text-align: left;" separator=" "/>
+        </xpath>
+        <xpath expr="//a" position="attributes">
+            <attribute name="class" remove="btn-lg" separator=" "/>
+        </xpath>
+    </template>
+
+    <!-- ======== PICTURE ======== -->
+    <template id="s_picture" inherit_id="website.s_picture" name="Vehicle s_picture">
+    <!-- Section -->
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="pt0 pb0 oe_img_bg" remove="pt48 pb24" separator=" "/>
+        </xpath>
+        <xpath expr="//section" position="attributes">
+            <attribute name="style" add="background-image: url(/web/image/website.s_picture_default_image); background-position: 50% 0;"/>
+        </xpath>
+        <!-- Layout -->
+        <xpath expr="//div[hasclass('container')]" position="replace">
+            <div class="container-fluid">
+                <div class="row">
+                    <div class="col-lg-12 oe_img_bg o_bg_img_center text-center d-flex flex-column" style="height:850px;">
+                        <div class="o_we_bg_filter" style="background-color: rgba(40,40,40,0.50)"></div>
```

Comment:
Why is there an oe_img_bg class etc + a filter element in this colummn? That's not something you can do via the editor